Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.6
+
+ADD . /app
+
+RUN cd app && sh docker/container-install.sh
+
+WORKDIR /app
+
+ENV MYSQL_HOST localhost
```

Comment:
The reason I did this was pretty selfish, I just wanted to get it working, and then make it better, so the next few iterations of the container on my fork is likely going to do away with the internal mysql server